Swift_Alert/Swift_Alert/MainTabView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ContentView.swift
+//  Swift_Alert
+//
+//  Created by coder on 16.01.2023.
+//
+
+import SwiftUI
+
+/// Главный таб бар
+struct MainTabView: View {
+    
+    // MARK: - @StateObject
+    @StateObject private var viewModel = PurchaseViewModel()
```

Comment:
```suggestion
    @StateObject private var viewModel = PurchaseViewModel()
```
private ниже var body